Source/WebKit/UIProcess/WebProcessCache.cpp

```diff
@@ -128,7 +152,7 @@ bool WebProcessCache::addProcess(std::unique_ptr<CachedProcess>&& cachedProcess)
     RELEASE_ASSERT(!registrableDomain.isEmpty());
 
     if (auto previousProcess = m_processesPerRegistrableDomain.take(registrableDomain))
-        WEBPROCESSCACHE_RELEASE_LOG("addProcess: Evicting process from WebProcess cache because a new process was added for the same domain", previousProcess->process().processID());
+        WEBPROCESSCACHE_RELEASE_LOG("addProcess: Evicting process from WebProcess cache because a new process was added for the same domain %{public}s", previousProcess->process().processID(), registrableDomain.string().utf8().data());
```

Comment:
I think we treat `registrableDomain` as `{private}`, because we don't want logging of origins visited.

Reply:
Thanks for reviewing! I will address these issues in the coming patch.